### ISSUE TITLE
Warning: Store + Typescript + increments\decrements

### ISSUE THREAD

**Issue by Th0rN13** [sveltejs/language-tools#170]
**Describe the bug**
Store with number type warned about increments\decrements
`$count++` -
`The operand of an increment or decrement operator must be a variable or a property access. ts(2357)`

**To Reproduce**
I use template https://github.com/dummdidumm/template

App.svelte
```
<script lang="ts">
  import { writable } from 'svelte/store';
  const count = writable<number>(0);
</script>

<button on:click={() => $count++}>add</button>
<button on:click={() => $count = 0}>clr</button>
{$count}
```

**Expected behavior**
No warnings

**System (please complete the following information):**
 - OS: Windows
 - IDE: VSCode
 - Plugin: "Svelte Beta"

**Comment by sw-yx** [comment]
interesting. haven't investigated, but what does typescript think $count is?

**Comment by Th0rN13** [comment]
On hover shows 
```
(alias) const count: Writable<number>
import count
```

**Comment by dummdidumm** [comment]
We have to check what `svelte2tsx` transforms this to. There is a  `__sveltets_store_get` function, maybe it gets in the way there.

**Comment by jasonlyu123** [comment]
It got transformed to `__sveltets_store_get($count)++`. Need to add a check for imcrement/decrement and change it to  `__sveltets_store_get($count)+ 1` or something else

**Comment by dummdidumm** [comment]
`$store += 1 ` will also throw then I guess.

**Comment by sw-yx** [comment]
@dummdidumm i also addressed `$store += 1` in #173. ended up taking more code than the original issue haha
